src/podping_hivewriter/podping_hivewriter.py

```diff
@@ -87,20 +89,20 @@ def __init__(
     async def _startup(self):
 
         try:
-            hive = await self.hive_wrapper.get_hive()
-            account = Account(self.server_account, blockchain_instance=hive, lazy=True)
+            self.lighthive_client = await get_automatic_node_selection(
+                self.lighthive_client
+            )
             settings = await self.settings_manager.get_settings()
-
             allowed = get_allowed_accounts(
-                settings.main_nodes, settings.control_account
+                self.lighthive_client, settings.control_account
             )
             # TODO: Should we periodically check if the account is allowed
             #  and shut down if not?
             if self.server_account not in allowed:
                 logging.error(
                     f"Account @{self.server_account} not authorised to send Podpings"
                 )
-        except AccountDoesNotExistsException:
+        except 45:
```

Comment:
I don't really know how to do error trapping properly and play nice with loops. Basic checks work.